### PR TITLE
Add code to check for determinism

### PR DESCRIPTION
**Pull request by greglow-sdu**
This article does a good job of describing determinism but lacks code that lets the reader determine whether or not a function is actually deterministic. I've added sample code to do that.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@greglow-sdu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/user-defined-functions/deterministic-and-nondeterministic-functions.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ Deterministic functions must be [schema-bound](user-defined-functions.md#SchemaB
 
 This article identifies the determinism of built-in system functions and the effect on the deterministic property of user-defined functions when it contains a call to extended stored procedures.
 
+### Determining if a function is deterministic
+
+You can check whether or not a function is deterministic by checking the "is_deterministic" object property for the function. The example below determines if the function Sales.CalculateSalesTax is deterministic.
```
Comment (rwestMSFT) [inline docs/relational-databases/user-defined-functions/deterministic-and-nondeterministic-functions.md:32]:
```suggestion
You can check whether a function is deterministic by querying the `is_deterministic` object property for the function. The example below determines if the function `Sales.CalculateSalesTax` is deterministic.

```

**Comment by rwestMSFT** [comment]
Thanks @greglow-sdu. I've made a suggestion. Once you approve that I'll get this merged.

**Comment by PRMerger15** [comment]
@greglow-sdu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by greglow-sdu** [comment]
Sounds good. Done !

Regards,

Greg

Dr Greg Low

1300SQLSQL (1300 775 775) office | +61 419201410 mobile
SQL Down Under | Web: https://sqldownunder.com<https://sqldownunder.com/> | About Greg:  https://about.me/greg.low

**Comment by rwestMSFT** [comment]
#sign-off

**Review by rwestMSFT — APPROVED** [review]

**Pull request merged**